README.md

```diff
@@ -0,0 +1,49 @@
+## Usage
+### Neovim
+This grammar can be setup within [Neovim](https://github.com/neovim/neovim/) to provide highlighting for the `.inp` files used in [TOPAS-Academic](http://www.topas-academic.net/). 
+It requires:
+- **Neovim 0.10** or newer.
+- Git
+- A C compiler
+1. Install [Nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter) using your preferred package manager. With [vim-plug](https://github.com/junegunn/vim-plug), for example, add this command to your `init.vim`:
+      ```vimscript
+      Plug 'nvim-treesitter/nvim-treesitter', {'do': ':TSUpdate'}
+      ```
+    then restart Neovim and run `:PlugInstall`
+  
+2. Add the following Lua snippet to `init.vim`:
+    ```lua
+    lua << EOF
```

Comment:
Are the `lua << EOF` & `EOF` bits relevant?

Reply:
I wasn't sure whether to include them or not, as my instructions assumed an `init.vim`. Forgetting Lua tags caused me a bit of confusion when I was testing out the instructions, but perhaps that's just because I hadn't used Vim before though. I've taken them out now.